lib/inspec/runner_rspec.rb

```diff
@@ -87,17 +87,17 @@ def report
       reporter.output_hash
     end
 
-    private
-
     # Empty the list of registered tests.
     #
     # @return [nil]
     def reset_tests
       @tests = RSpec::Core::World.new
       # resets "pending examples" in reporter
-      RSpec.configuration.reset
+      # RSpec.configuration.reset
```

Comment:
I restructured this.  Essentially - I wanted to reset_tests in the test_collector without resetting the output configuration and json formatter stuff.  But I found a more explicit way to do that.